v21.1/restore.md

```diff
@@ -354,6 +354,13 @@ job_id             |  status   | fraction_completed | rows | index_entries | byt
 
 By default, tables and views are restored to the database they originally belonged to. However, using the [`into_db` option](#into_db), you can control the target database.
 
+First, create the new database that you'll restore the table or view into:
+
+{% include_cached copy-clipboard.html %}
+~~~ sql
+> CREATE DATABASE newdb;
+~~~
+
```

Comment:
Nit: Maybe, before the `RESTORE` statement, you could have a short introductory sentence. So, since you precede the first statement with "First"... something like:

> Next, restore the table into the newly create database with `into_db`:
> 
> {% include_cached copy-clipboard.html %}
> ~~~ sql
> RESTORE bank.customers FROM LATEST IN 's3://{bucket_name}?AWS_ACCESS_KEY_ID={key_id}&AWS_SECRET_ACCESS_KEY={access_key}'
> WITH into_db = 'newdb';
> ~~~